docs/src/components/AppCacheBanner/index.css

```diff
@@ -1,6 +1,6 @@
 .banner {
   position: fixed;
-  position: sticky;
+  position: sticky; /* stylelint-disable-line */
```

Comment:
Hello. I hadn't considered that use case when enabling the `declaration-block-no-duplicate-properties` rule. I had assumed all fallbacks would be handled by tooling.

There's an appropriate option that to make the rule more permissive for this situation: [`ignore: ["consecutive-duplicates-with-different-values"]`](https://github.com/stylelint/stylelint/tree/master/src/rules/declaration-block-no-duplicate-properties#ignore-consecutive-duplicates-with-different-values).

I've created [an issue](https://github.com/stylelint/stylelint-config-standard/issues/42) to discuss using that option by default in the standard config.